pkg/operator/ceph/csi/secrets_test.go

```diff
@@ -124,3 +136,161 @@ func Test_deleteOwnedCSISecretsByCephCluster(t *testing.T) {
 		t.Errorf("expected unowned secret %q to still exist", CsiCephFSProvisionerSecret)
 	}
 }
+
+func TestSortCsiClientName(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    []string
+		want    []string
+		wantErr bool
+	}{
+		{
+			name: "testName",
+			args: keys,
```

Comment:
For unit tests, more proximity and explicitness is usually better.

```suggestion
			args: []string{"client.csi-rbd-node.10", "client.csi-rbd-node.1", "client.csi-rbd-node.4", "client.csi-rbd-node.2", "client.csi-rbd-node"},
```